src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -142,6 +142,8 @@ private void transferEnchantments(ItemStack item, ItemStack book, Map<Enchantmen
     private boolean isDisenchantable(@Nullable ItemStack item) {
         if (item == null) {
             return false;
+        } else if (hasIgnoredLore(item)) {
+            return false;
```

Comment:
This is a rather heavy method call as it invokes `.getItemMeta()` so it should only happen further down.
The book check was put in place to aid performance, changing this order will render this performance improvement useless.